yaksh/templates/yaksh/add_course.html

```diff
@@ -67,7 +67,7 @@
 
         <center>
             <button class="btn btn-success btn-lg" type="submit" id="submit" name="course">
-                Save
+                <i class="fa fa-save"></i>&nbsp;Save
```

Comment:
avoid using &nbsp.

Reply:
I feel this is better and more html rather than using manual spacing.